File: 2148-minimum-number-of-moves-to-seat-everyone/2148-minimum-number-of-moves-to-seat-everyone.cpp
#include <cmath>
class Solution {
public:
    int minMovesToSeat(vector<int>& seats, vector<int>& students) {
        int n = seats.size();
        sort(seats.begin(),seats.end());
        sort(students.begin(),students.end());
        int total = 0;
        for(int i = 0; i < n; i++)
        {
            total+= abs(seats[i]-students[i]);
        }
        return total;
    }
};

File: 2148-minimum-number-of-moves-to-seat-everyone/2148-minimum-number-of-moves-to-seat-everyone.py
class Solution:
    def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
        #sorting each array
        seats.sort()
        students.sort()

        #finding the difference for each value
        count = 0
        for i in range(len(seats)):
            count += abs(seats[i] - students[i])
        return count